src/test/kotlin/org/openbroker/se/privateunsecuredloan/TestObjectsJson.kt

```diff
@@ -114,13 +114,15 @@ object TestObjectsJson {
 					"loanAmount": 4000,
 					"monthlyPayment": 22,
 					"shouldRefinance": true,
+                    "refinanceAmount": 4000,
```

Comment:
Whitespace is inconsistent with surrounding lines. Maybe change other lines to spaces or use tabs for consistency. Same for lines below.